src/handlers/beforeReady.js

```diff
@@ -1,3 +1,9 @@
-export default async function beforeReady() {
-  // No-op by default
+import {
+  identifyAuthenticatedUser,
+  sendPageEvent,
+} from '@edx/frontend-analytics';
+
+export default async function beforeReady(app) {
+  identifyAuthenticatedUser(app.authenticatedUser.userId);
+  sendPageEvent();
```

Comment:
1. Does that mean that the whole thing needs to be overridden for apps that have multiple pages?
2. Does that mean this is missing from payment, or are you just moving this into frontend-base?

Reply:
1. An app with multiple pages will need to call sendPageEvent when it switches pages - given that this code is just about initialization, there's not much we can do to help that here.  (It would have been the same before frontend-base existed)

2. It's happening in payment production/always has been.  So that's good.  It's currently missing from my branch of payment, though - I missed it when I was building frontend-base.  And by the same token, it's currently missing from profile in production.